src/mlpack/bindings/cli/map_parameter_name.hpp

```diff
@@ -53,14 +53,16 @@ std::string MapParameterName(
 }
 
 /**
- * Map the parameter name to the name seen by boost::program_options.
- *
+ * Map the parameter name to the name seen by CLI instead of boost.
+ *  
+ * Not sure if we will keep this funtion since there is not boost 
```

Comment:
Just wanted to put a comment here, let's make sure that before merge we find out if `MapParameterName` is still necessary. :+1:

Reply:
Actually, I do think that this will remain necessary.  The functionality of `MapParameterName` is to turn the argument names associated with a matrix or model parameter into ones with `_file` appended.  So, e.g., if you specify `PARAM_MATRIX("input", ...)`, the CLI bindings actually need to have the argument name `input_file` since it's the path to the file that's being input.  That's the same even when we change from boost::program_options to CLI11, so we can probably leave it here.